api/features/import_export/permissions.py

```diff
@@ -56,3 +57,28 @@ def has_permission(self, request: Request, view: ListAPIView) -> bool:
         # The user will only see environment feature exports
         # that the user is an environment admin.
         return request.user.has_project_permission(VIEW_PROJECT, project)
+
+
+class FeatureImportListPermissions(IsAuthenticated):
+    def has_permission(self, request: Request, view: GenericViewSet) -> bool:
+        if not super().has_permission(request, view):
+            return False
+
+        project = Project.objects.get(id=view.kwargs["project_pk"])
+        # The user will only see environment feature imports
+        # that the user is an environment admin.
+        return request.user.has_project_permission(VIEW_PROJECT, project)
+
+
+class FeatureImportRetrievePermissions(IsAuthenticated):
+    def has_permission(self, request: Request, view: GenericViewSet) -> bool:
+        if not super().has_permission(request, view):
+            return False
+
+        feature_import = view.get_queryset().filter(pk=view.kwargs["pk"]).first()
+
+        if not feature_import:
+            return False
```

Comment:
I'd say it's perhaps better to use `get_object_or_404` here?

Reply:
Would have been better but no longer necessary as that permission is now deleted.